.env.example

```
@@ -1,12 +1,10 @@
-SNUTT_API_BASE=https://snutt-api.wafflestudio.com 
-SNUTT_API_KEY=YOUR_API_KEY 
-SNUTT_ACCESS_TOKEN=YOUR_ACCESS_TOKEN 
+# Local sugang dataset path (optional)
+SNUTT_LOCAL_DATA_DIR=data/sugang
 
-# 캐시/레이트 리밋 조정
-SNUTT_CACHE_TTL_SECONDS=1800      #강의 목록 캐시 TTL, 기본 1800초
-SNUTT_RATE_LIMIT_WINDOW_MS=60000  #IP별 윈도우, 기본 60000ms
-SNUTT_RATE_LIMIT_MAX=N            #윈도우 내 최대 요청, 기본 30
+# Cache / rate-limit
+SNUTT_CACHE_TTL_SECONDS=1800      # lecture list cache TTL, default 1800s
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)